index.js

```diff
@@ -1,4 +1,21 @@
-const permissions = require('bindings')('permissions.node')
+const nonMacResponse = () => undefined
+const stub = {
+  askForCalendarAccess: nonMacResponse,
+  askForContactsAccess: nonMacResponse,
+  askForFoldersAccess: nonMacResponse,
+  askForFullDiskAccess: nonMacResponse,
+  askForRemindersAccess: nonMacResponse,
+  askForCameraAccess: nonMacResponse,
+  askForMicrophoneAccess: nonMacResponse,
+  askForPhotosAccess: nonMacResponse,
+  askForSpeechRecognitionAccess: nonMacResponse,
+  askForScreenCaptureAccess: nonMacResponse,
+  askForAccessibilityAccess: nonMacResponse,
+  getAuthStatus: nonMacResponse,
+}
+
+const os = require('os')
+const permissions = os.platform() === 'darwin' ? require('bindings')('permissions.node') : stub
```

Comment:
nit but let's use either `os.platform()` or `process.platform` everywhere?

Reply:
I have a reason for the difference here. I actually am not sure if `process.platform` could be considered "dead code" by a bundler at compile time in the same manner that `process.env.NODE_ENV === 'development'` could be stripped from a prod bundle.

I was keeping the require's in the tests to a minimum while using a runtime eval for the conditional binding. I can update the test to use `os.platform()`